{"links": {"self": {"href": "data/repositories/osrf/swarm/issues/15/comments/23133339.json"}, "html": {"href": "#!/osrf/swarm/issues/15#comment-23133339"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/swarm/issues/15.json"}}, "type": "issue", "id": 15, "repository": {"links": {"self": {"href": "data/repositories/osrf/swarm.json"}, "html": {"href": "#!/osrf/swarm"}, "avatar": {"href": "data/bytebucket.org/ravatar/{06a294a4-4254-4f63-b7b3-e5768577fb96}ts=default"}}, "type": "repository", "name": "swarm", "full_name": "osrf/swarm", "uuid": "{06a294a4-4254-4f63-b7b3-e5768577fb96}"}, "title": "lost person starts at high altitude"}, "content": {"raw": "@esquires, do you still have this issue after merging pull request #73?", "markup": "markdown", "html": "<p>@esquires, do you still have this issue after merging <a href=\"#!/osrf/swarm/pull-requests/73/lost-person-controller-renamed-to-what-is\" rel=\"nofollow\" class=\"ap-connect-link\">pull request #73</a>?</p>", "type": "rendered"}, "created_on": "2015-11-06T22:47:57.528756+00:00", "user": {"display_name": "Carlos Ag\u00fcero", "uuid": "{da8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bda8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a%7D"}, "html": {"href": "https://bitbucket.org/%7Bda8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/692bf15758111acaddae4da15a47f9e5d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsCA-0.png"}}, "nickname": "caguero", "type": "user", "account_id": "557058:4ded1ddf-947e-4154-bbd1-3dba24f1bdbd"}, "updated_on": null, "type": "issue_comment", "id": 23133339}